helm/prometheus-rules/templates/recording-rules/service-level.rules.yml

```diff
@@ -281,7 +281,7 @@ spec:
         provider: aws
       record: raw_slo_errors
       # -- 99% availability
-    - expr: "vector((1 - 0.99))"
+    - expr: label_replace(group(cloudprovider_aws_api_request_duration_seconds_count{app=~"kube-controller-manager|kube-scheduler"}) by (cluster_id, app), "service", "$1", "app", "(.*)") * 0 + 1 - 0.99
```

Comment:
Keeping 99% availability but adding the missing and mandatory `service` label

![image](https://user-images.githubusercontent.com/6536819/151942462-caeac83f-9cd3-4866-a118-1cb6547130a1.png)